packages/web-components/fast-components/src/calendar/index.ts

```diff
@@ -1,10 +1,26 @@
 import {
-    Calendar,
     calendarTemplate,
     CalendarTitleTemplate,
+    Calendar as FoundationCalendar,
 } from "@microsoft/fast-foundation";
+import { fillColor, neutralLayerFloating } from "../design-tokens";
 import { CalendarStyles as styles } from "./calendar.styles";
 
+/**
+ * The FAST listbox class
+ * @public
+ */
+export class Calendar extends FoundationCalendar {
+    /**
+     * @internal
+     */
+    public connectedCallback(): void {
+        super.connectedCallback();
+
+        fillColor.setValueFor(this, neutralLayerFloating);
```

Comment:
Let's use the layer recipe instead, treating it like a card.
```suggestion
        fillColor.setValueFor(this, neutralFillLayerRest);
```